shell/platform/darwin/ios/framework/Source/FlutterView.h

```diff
@@ -9,10 +9,25 @@
 
 #include <memory>
 
+#include "flutter/fml/memory/weak_ptr.h"
+#include "flutter/shell/common/shell.h"
 #include "flutter/shell/platform/darwin/ios/ios_surface.h"
 
+@protocol FlutterScreenshotDelegate <NSObject>
```

Comment:
Do you plan to use this elsewhere?  If not, it's clear to me what advantage this provides over passing the `FlutterViewController` down directly like you did in the previous version.

Reply:
No, but it feels pretty wrong to pass a view controller directly into a view.  I was looking at passing in just the shell, and after discussion with @chinmaygarde decided to go with a delegate - this way we avoid trying to share a pointer to the shell, and we avoid tying the ViewController implementation details to the View.